### DISCUSSION TITLE
Draft Issues

### DISCUSSION THREAD

**Discussion by Malix-Labs** [community/community#19498] [Projects and Issues]
# Draft Issues

Being able to create draft issues (not only for project, but as a whole, like [draft PR](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests#draft-pull-requests))

## Links

- [idea source](https://github.com/github-community/community/discussions/16610#discussioncomment-3026682)

**Comment by alvaromartmart** [comment]
I think this makes a lot of sense.
We can create "drafts" from a project, but then you cannot use Issue templates, for instance, which is a deal breaker for me

**Comment by jeritiana** [comment]
I think this also has been discussed in https://github.com/isaacs/github/issues/1648 with already 49 👍 upvotes.

**Reply by Malix-Labs** [reply]
Yep, but it is not in the GitHub's official discussion

**Comment by itcarroll** [comment]
Today I needed to move a bunch of drafts to a new project. There was no way to do this. Had to re-create and delete. If the draft was an issue, it would have been easy.

How would a draft issue actually be different from an issue? I think it's all about visibility. A draft issue's visibility would be controlled by what project(s) it belongs to. It would not appear in a repository's issue list to anyone who could not see it in a project.
